src/integrationTest/java/org/opensearch/test/framework/certificate/Certificates.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.test.framework.certificate;
+
+/**
+ * Contains static certificates for the test cluster. 
+ * Note: This is WIP and will be replaced by classes
```

Comment:
I would like to see this resolved before this PR is merged, is this planned?

Reply:
Actually, we planned to provide this as a separate PR to avoid increasing the current PR volume. Additionally code associated with certificate generation can be reviewed better in independent PR and it is easier to introduce corrections in smaller PRs.

Reply:
Could you create an issue for this follow up so we don't lose it?

Reply:
Sorry for the late answer. I created an issue: https://github.com/opensearch-project/security/issues/2057